Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/TypeTester.cs

```diff
@@ -1,41 +1,95 @@
-﻿using Newtonsoft.Json.Linq;
+﻿using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Text;
 using NUnit.Framework;
 
 namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
 {
     public abstract class TypeTesterBase
     {
-        protected virtual void ConfigureSettings(JsonSerializerSettings settings)
+        protected readonly JsonSerializer _serializer;
+
+        static TypeTesterBase()
+        {
+            UnityConverterInitializer.ShouldAddConvertsToDefaultSettings = false;
+            UnityConverterInitializer.DefaultUnityConvertersSettings.Formatting = Formatting.None;
+        }
+
+        protected TypeTesterBase()
+        {
+            _serializer = JsonSerializer.Create(UnityConverterInitializer.DefaultUnityConvertersSettings);
+        }
+
+        [SetUp]
+        public void SetUpSerializer()
+        {
+            ConfigureSerializer(_serializer);
+        }
+
+        protected virtual void ConfigureSerializer(JsonSerializer serializer)
         {
             // No settings changes by default
         }
 
-        protected JsonSerializerSettings GetSettings()
+        protected string Serialize(object? anonymous)
         {
-            JsonSerializerSettings settings = UnityConverterInitializer.GetDefaultUnitySettings();
-            settings.Formatting = Formatting.None;
-            ConfigureSettings(settings);
-            return settings;
+            return Serialize(anonymous, _serializer);
         }
 
-        protected TypeTesterBase()
+        protected static string Serialize(object? anonymous, JsonSerializer serializer)
```

Comment:
Variable name `anonymous` is a bit misleading. This method is used to just facade serialization of any type, not only anonymous types.

Reply:
Changed to "value"